### ISSUE TITLE
Quick info does not show uo.

### ISSUE THREAD

**Issue by yekanchi** [wmjordan/Codist#312]
I the latest version (7.8) not only quick info does not show up. but by disabling it the default quick info of the visual studio does not show up anymore.
had to disable the whole `Codist` extension.

Visual Studio Version: 17.8.5 and 17.9 preview (I checked both)

**Comment by wmjordan** [comment]
Please check if the "Hide Quick Info until Shift key is press" option is accidentally checked.
<img width="574" alt="image" src="https://github.com/wmjordan/Codist/assets/12182730/9d75de86-b9be-4453-b5e6-77008a848cdd">

**Comment by yekanchi** [comment]
> Please check if the "Hide Quick Info until Shift key is press" option is accidentally checked. <img alt="image" width="574" src="https://private-user-images.githubusercontent.com/12182730/298832743-9d75de86-b9be-4453-b5e6-77008a848cdd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYwMDc0OTksIm5iZiI6MTcwNjAwNzE5OSwicGF0aCI6Ii8xMjE4MjczMC8yOTg4MzI3NDMtOWQ3NWRlODYtYjliZS00NDUzLWI1ZTYtNzcwMDhhODQ4Y2RkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIzVDEwNTMxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhNzI1Yzk2MjViODMxYTVlZGZmNmRjMmRkNjMwNGY4ZDY2MzU4ZjUzZjFjOGY2OWQ5YmE0OTIxNjhlOGVhZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6Q3cNvy7gKoYOOuZGFMPO2LAB8Ykecefpbz_4tiYu9s">

yes it was the case, and it's weired it happend in both of versions

**Comment by wmjordan** [comment]
Codist shares config data across VS versions. Thus it happened in your both versions.

**Comment by wmjordan** [comment]
Thank you for helping find out a bug.

The Quick Info controller was not controlled by the feature control, thus it was still in effect, when you disabled the Super Quick Info feature. I will fix this in the next release.
